src/app/login-page/login-page.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="login-container">
+  <div class="title-text"><h1>Login</h1></div>
+
+  <form
+    class="login-box"
+    [formGroup]="loginForm"
+    (ngSubmit)="onLoanFormSubmit()"
+    (reset)="onLoanFormReset()"
+  >
+    <label for="login">Login: </label>
+    <input type="email" id="login" formControlName="login" />
+
+    <label for="password">Password: </label>
+    <input type="password" id="password" formControlName="password" />
```

Comment:
Let's see if we can change these to Angular Material today - just to align the styling.